content/en/docs/components/pipelines/v1/sdk/connect-api.md

```diff
@@ -152,146 +152,186 @@ and give them access to submit pipelines in their own namespace.
 <summary>Click to expand</summary>
 <hr>
 
-The process to authenticate the Pipelines SDK from outside the cluster in multi-user mode will vary by distribution:
+When running the Kubeflow Pipelines SDK from outside a multi-user Kubeflow cluster (e.g. on a laptop), the `kfp.Client()` will need to be initialized with credentials.
 
-* [Kubeflow on Google Cloud](/docs/distributions/gke/pipelines/authentication-sdk/#connecting-to-kubeflow-pipelines-in-a-full-kubeflow-deployment)
-* [Kubeflow on AWS](/docs/distributions/aws/pipeline/#authenticate-kubeflow-pipeline-using-sdk-outside-cluster)
-* [Kubeflow on Azure](https://awslabs.github.io/kubeflow-manifests/docs/component-guides/pipelines/)
-* [Kubeflow on IBM Cloud](/docs/distributions/ibm/pipelines/#2-authenticating-multi-user-kubeflow-pipelines-with-the-sdk)
```

Comment:
Understanding that distribution section has been moved out of the `kubeflow.org`, I am leaning towards keeping this redirection using `updated links to each distribution website`. We can move this highlighted part to the bottom of this page around `## Next Steps` section, so the generic connection approach is higher priority on the page.

Reply:
@zijianjoy the problem is deciding which distributions get to be listed here, (and which page we would even link to).

I think the "tip" which links back to the "installing kubeflow" page should be sufficient:

![Screenshot 2023-08-09 at 12 16 40](https://github.com/kubeflow/website/assets/5735406/475ae731-ad58-4a71-8c71-5e3c2208ee22)

Reply:
I would suggest to move this change to another PR, so I can help merging the `dex authentication` portion quicker, which aligns with the title of this PR (update kfp sdk "dex authentication" example)

Reply:
+1 on what @zijianjoy said. @thesuperzapper please focus on the area you're trying to update without disturbing the rest.

Reply:
I think this update (to remove these links) is relatively important, as the links are to distribution docs that no longer exist as part of https://github.com/kubeflow/website/issues/3074.

Furthermore, as part of https://github.com/kubeflow/website/issues/3074, we try not have any distribution-specific docs on the main website.

_PS: The reason we are including the dex example, is because that is the auth that the upstream `kubeflow/manifests` uses._